code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -623,6 +672,7 @@ var/global/list/robot_modules = list(
 	src.modules += new /obj/item/weapon/form_printer(src)
 	src.modules += new /obj/item/weapon/gripper/paperwork(src)
 	src.modules += new /obj/item/weapon/hand_labeler(src)
+	src.modules += new /obj/item/weapon/folder/red(src) // for any remarks

```

Comment:
Won't this just consume paper into it, as in, you're never able to remove it? Since you've nuked most paperwork stuff, please also remove this.